src/ctelnet.cpp

```diff
@@ -808,6 +808,8 @@ void cTelnet::sendTelnetOption(char type, char option)
 void cTelnet::replyFinished(QNetworkReply* reply)
 {
     mpProgressDialog->close();
+    packageDownloadReply->abort();
```

Comment:
Isn't the contents of `packageDownloadReply` going to be the same as `reply`? 😕 

Using `QNetworkReply::abort()` on either AFTER the point at which the slot `cTelnet::replyFinished(QNetworkReply* reply)` would have been called seems pointless... 😖